src/lib/Components/Bidding/Screens/__tests__/SelectMaxBid-tests.tsx

```diff
@@ -33,9 +48,63 @@ const SaleArtwork = {
   ],
 }
 
+let fakeNavigator: FakeNavigator
+let fakeRelay: {
+  refetch: jest.Mock
+}
+
+beforeEach(() => {
+  fakeNavigator = new FakeNavigator()
+  fakeRelay = {
+    refetch: jest.fn(),
+  } as any
+})
+
 it("renders properly", () => {
   const component = renderer
-    .create(<SelectMaxBid me={Me} sale_artwork={SaleArtwork} navigator={jest.fn() as any} />)
+    .create(
+      <SelectMaxBid me={Me} sale_artwork={SaleArtwork} navigator={fakeNavigator as any} relay={fakeRelay as any} />
+    )
     .toJSON()
   expect(component).toMatchSnapshot()
 })
+
+it("shows a spinner while fetching new bid increments", () => {
+  const component = renderer.create(
+    <SelectMaxBid me={Me} sale_artwork={SaleArtwork} navigator={fakeNavigator as any} relay={fakeRelay as any} />
+  )
+
+  component.root.instance.setState({ isRefreshingSaleArtwork: true })
```

Comment:
I was thinking about mocking the `refresh` so it accepts the same signature, including the callback, and have the state set by the callback that is defined in the component instead of explicitly setting it here. But with the implementation I can think of the mock is going to be too complicated and the test mostly verifies the mock rather than the component so an overkill, unless there is a smarter way to do that 😄

Reply:
Just noticed you have such e mock for the last test so all good 😄